src/java/org/apache/cassandra/db/marshal/DecimalType.java

```diff
@@ -168,6 +172,87 @@ public int next()
         };
     }
 
+    @Override
+    public <V> V fromComparableBytes(ValueAccessor<V> accessor, ByteSource.Peekable comparableBytes, ByteComparable.Version version)
```

Comment:
One of my favorite pieces of code that I've written... Would love seeing it in the open - thank you for putting it there!